docs/configuration.md

```diff
@@ -2985,6 +2985,21 @@ External users can query the static sql config values via `SparkSession.conf` or
 </tr>
 </table>
 
+### REPL
+
+<table class="table">
+<tr><th>Property Name</th><th>Default</th><th>Meaning</th><th>Since Version</th></tr>
+<tr>
+  <td><code>spark.repl.inactivityTimeout</code></td>
+  <td>0s</td>
+  <td>
+    Idle timeout for user input in spark-shell. The shell and the SparkContext will be closed if no new lines are
+    entered for the specified amount of time. The default value (0s) disables the feature.
```

Comment:
Let's mention the time unit is milliseconds here. 
The default value is 0ms (though the value is the same as 0s)

Reply:
Thanks for your review! I have pushed a change where I added that the value is milliseconds based. Please let me know if it looks good to you know or if the description should be further updated.